src/content/packages-and-plugins/swift-package-manager/for-app-developers.md

```diff
@@ -100,7 +100,13 @@ To undo this migration:
 
 1. [Turn off Swift Package Manager][].
 
-1. Open your app (`ios/Runner.xcworkspace` or `ios/Runner.xcworkspace`) in
+1. Clean your project:
+
+   ```sh
+   flutter clean
+   ```
```

Comment:
This isn't strictly necessary but it results in much better error messages if you miss a step.